### ISSUE TITLE
Access to starting point of each mini-project

### ISSUE THREAD

**Issue by mbluemer** [learncodeacademy/react-js-tutorials#20]
The best way to learn from a tutorial series like this is to follow along and code with the videos. This repo currently has no way to start the mini-projects from where the videos start and each project is different enough from the others that following along from scratch is difficult.

It would be useful to either have a branch for the starting points, have separate directories for starting points, or even have instructions within each mini-project directory for rolling back commits to a starting point.

**Comment by willrstern** [comment]
@mbluemer I agree with you on that.  It'll be a little bit of work to back-cycle code to the starting points, but I think it's a good goal for this repo to have.

**Comment by pedromartinez11** [comment]
For "2-react-router", I reset my local code to commit f923c11ae2dfa921f4a7f7d25a47466ed1495eba by doing:
git reset --hard f923c11ae2dfa921f4a7f7d25a47466ed1495eba

I think you can do the same thing for the other parts.

**Comment by arumals** [comment]
It would be awesome to have individual branches for each of the starting points.
